precompiles/assets-erc20/src/tests.rs

```diff
@@ -649,7 +649,7 @@ fn transfer_from_non_incremental_approval() {
 						value: 500.into(),
 					},
 				)
-				.expect_cost(36390756u64)
+				.expect_cost(46033756u64)
```

Comment:
Why these increases?

Reply:
Because weight for these extrinsic increased 

https://github.com/paritytech/substrate/blob/7a4e5163091384c4c10b6d76f5cb80dac0834f38/frame/assets/src/weights.rs#L253